sklearn/linear_model/_bayes.py

```diff
@@ -464,6 +470,12 @@ class ARDRegression(RegressorMixin, LinearModel):
         Independent term in decision function. Set to 0.0 if
         ``fit_intercept = False``.
 
+    X_offset_ : float
+        if normalized, offset subtracted for centering data to mean zero
+
+    X_scale_ : float
+        if normalized, scaling parameter for division
```

Comment:
Do we need to add the `.. versionadded:: 0.24` directive for those attributes? It's a bugfix, they should always had been there so I am not sure.